R/Organism.R

```diff
@@ -806,12 +808,12 @@ setMethod("growth_par", "Bac", function(object, population, j, fbasol, tstep){
 #' arena <- addOrg(arena,bac,amount=10) #add 10 organisms
 #' arena <- addSubs(arena,40) #add all possible substances
 #' chemotaxis(bac,arena,1)
-setGeneric("chemotaxis", function(object, population, j){standardGeneric("chemotaxis")})
+setGeneric("chemotaxis", function(object, population, j, chemo){standardGeneric("chemotaxis")})
```

Comment:
Is there a reason why you want to add an aditional function argument? I think it is more simple to leave the information in `object@chem`.

Reply:
Yes there is the reason: Let's imagine c("EX_o2(e)", "EX_co2(e)") the chemo vector. In line 887 it uses every v element of the vector one y one.
Line 887
 chemo <- object@chem[[v]]
 chemo_pos <- chemotaxis(object, arena, j, chemo)

So the function takes the object@chem[[v]] not the object@chem. For stylistic reasons, I created a new parameter chemo.

Reply:
Ok, I see. Yes, it needs to be there.